index.html

```diff
@@ -250,15 +256,16 @@ <h3>Ecosystem Overview</h3>
         </dd>
         <dt><a>verifier</a></dt>
         <dd>
-A role an <a>entity</a> may perform by receiving one or more
-<a>verifiable credentials</a> for processing.
+A role an <a>entity</a> may perform by requesting and receiving a
+<a>verifiable presentation</a> that proves the holder possesses the required verifiable credentials with certain characteristics.
 Examples of verifiers include employers, security personnel, and websites.
         </dd>
         <dt><a>identifier registry</a></dt>
         <dd>
-A role a system may perform by mediating the creation and verification of
-<a>subject</a> identifiers. Examples of identifier registries include
-corporate employee databases, government ID databases, and distributed ledgers.
+Mediates the creation and verification of <a>issuer</a> identifiers, keys and
```

Comment:
We should stick with the "role" language for consistency, IMO. Also, this stripped out some of the language talking about identifiers for subjects, not just issuers. That these won't be used in all configurations doesn't mean we should not mention it for the ones where they are. We should have some language in here that covers both cases.

Reply:
Changed the language